user-relations/README.md

```diff
@@ -1 +1,17 @@
-# User Relations
\ No newline at end of file
+# User Relations
+
+```json
+{
+    "$schema": "sia://skystandards.hns/v1/userRelations.schema.json", // Type definition with spec version number (1)
+    "userId":"d448f1562c20dbafa42badd9f88560cd1adb2f177b30f0aa048cb243e55d37bd", // This is the global userId used by SkyID, not the app-specific public key (To ensure interoperability with other skapps)
+    "relationType": "following", // one of ["following","followers","blocked"]
```

Comment:
After a bit of a chat on discord I think I'm coming around to this data structure. Of course, actually storing the relation type inside the file that is already sorted into "folders" of different types is a tad redundant, but it's not a big deal and may be helpful in edge cases so that's cool.

Of course there's also the issue of using JSON in the first place for something that could need to scale to millions of lines, but that can be tackled at some future point with stuff like gun(see here: https://github.com/Fluffy9/Zenbase) or the upcoming SkySQL by DaWe. I'd really worry about this later though, there are many other things that need to be tackled before getting to that scale is possible anyway.